src/main/java/frc/robot/Ports.java

```diff
@@ -1,7 +1,7 @@
 package frc.robot;
 
 public final class Ports {
-    public static class ExampleSubsystem {
+    public static class ExampleSubsystem{
```

Comment:
Why would you remove the space?

Reply:
Is that important? Is that really important?